liqd_product/templates/footer.html

```diff
@@ -3,14 +3,25 @@
 <footer class="footer">
     <div class="l-wrapper">
         <nav class="navi">
+            {%  with pages=settings.liqd_product_cms_settings.ImportantPages %}
             <ul class="navi__list">
+                {% if pages.imprint %}
                 <li class="navi__item">
-                    <a href="#" class="navi__link">Impressum</a>
```

Comment:
just realized, that we should probably make Impressum etc translateable. i'll add that later